vllm/model_executor/models/llama.py

```diff
@@ -189,8 +192,10 @@ def __init__(
             cache_config=cache_config,
             quant_config=quant_config,
         )
+        # For CUDA devices and Navi4x, attn_fp8_out will be set to false.
         self.attn_fp8_out = envs.VLLM_USE_ROCM_CUSTOM_PAGED_ATTN_FP8_OUT \
                             and is_hip() \
+                            and not is_navi4x() \
```

Comment:
Fused attention kernel outputting fp8 could work on Navi

Reply:
Let's skip for now, but keep in mind as this will also become relevant for MI350 and later